lib/ansible/modules/cloud/amazon/code_commit.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Shuang Wang <ooocamel@icloud.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: code_commit
+version_added: "2.8"
+short_description: Manage repositories in AWS CodeCommit
+description:
+  - Supports creation and deletion of CodeCommit repositories.
+  - See U(https://aws.amazon.com/codecommit/) for more information about CodeCommit.
+author: Shuang Wang (@ptux)
+
+requirements:
+  - botocore
+  - boto3
+  - python >= 2.6
+
+options:
+  name:
+    description:
+      - name of repository.
+    required: true
+  comment:
+    description:
+      - description or comment of repository.
+    required: false
+  state:
+    description:
+      - Specifies the state of repository.
+    required: true
+    choices: [ 'present', 'absent' ]
+
+extends_documentation_fragment:
+  - aws
+  - ec2
+'''
+
+RETURN = '''
+result:
```

Comment:
This isn't really good enough. 

You can use hacking/return_skeleton_generator.py to create much more thorough return documentation